### ISSUE TITLE
Confused about the normal direction

### ISSUE THREAD

**Issue by EAST-J** [autonomousvision/monosdf#66]
Hi, thx for your great work.
I want to add the normal cues to the Neus, and I am little confused about the normal direction. In the Neus, they plot the normal map by
`normal_img = np.concatenate(out_normal_fine, axis=0)`
`rot = np.linalg.inv(self.dataset.pose_all[idx, :3, :3].detach().cpu().numpy())`
`normal_img = (np.matmul(rot[None, :, :], normal_img[:, :, None]).reshape([H, W, 3, -1]) * 128 + 128).clip(0, 255)`
It seems that they transform the normal map to world coordinate. I wonder If I use the monocular normal predicted by omnidata model, do I need to do some preprocess?

**Comment by niujinshuchong** [comment]
Hi, you could apply the loss in the local image coordinate system or in the world coordinate system as long as the rendered normal and monocular normal are in the same coordinate system. The omnidata monocular normal is in the local image coordinate system.
